app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -56,6 +56,19 @@ echo
   "connectefi #{connectefi_option}" if connectefi_option
 %>
 
+if [ "${lockdown}" == "y" ]; then
+
+  if [ "${default}" == "local" ]; then
+    set default=next_bootdevice
+  fi
+
+menuentry 'Booting from next boot device' --id next_bootdevice {
```

Comment:
Can you fix the indentation (2 spaces) of menuentry and the lines below as it will only exist if lockdown=y?

Reply:
You mean to insert whole menuentry block by two spaces?

What is the rule of thumb for rendered files which do not care about indentation? Using indentation like you would write code or a script resulting in "unreadable" rendered files with wild indentations?

Or to ignore any indentations and try to produce readable rendered files with sane indentations?

Or a mix of both, where if- and loop-conditions are indented but not the actual lines which will be rendered?

Reply:
Well, in this case, the rendered output would require the indentation.
Without indentation (current state of the PR):
```
if [ "${lockdown}" == "y" ]; then

  if [ "${default}" == "local" ]; then
    set default=next_bootdevice
  fi

menuentry 'Booting from next boot device' --id next_bootdevice {
  echo "SecureBoot is enabled, attempting next boot device..."
  sleep 2
  exit 1
}
fi
```

With the suggested indentation:
```
if [ "${lockdown}" == "y" ]; then

  if [ "${default}" == "local" ]; then
    set default=next_bootdevice
  fi

  menuentry 'Booting from next boot device' --id next_bootdevice {
    echo "SecureBoot is enabled, attempting next boot device..."
    sleep 2
    exit 1
  }
fi
```

Reply:
> the rendered output would require the indentation.

No. GRUB2 configuration doesn't require indentation, you can indent every line as you want (like Bash).

Anyway, I looked up my local `/boot/grub2/grub.cfg` (openSUSE Leap) which contains a lot of conditions and they use indeed indentation. So I will adapt to your suggestion.